xs/src/libslic3r/Model.cpp

```diff
@@ -20,6 +20,9 @@ Model::Model(const Model &other)
     this->objects.reserve(other.objects.size());
     for (ModelObjectPtrs::const_iterator i = other.objects.begin(); i != other.objects.end(); ++i)
         this->add_object(**i, true);
+
+    // copy metadata
+    this->metadata = other.metadata;
```

Comment:
The reasoning is that it should be initialized to something for a non-3mf load (I don't know if you tested this).